composer.json

```diff
@@ -3,7 +3,7 @@
     "license": "MIT",
     "type": "project",
     "description": "Symfony Demo Application",
-    "minimum-stability": "stable",
+    "minimum-stability": "dev",
```

Comment:
should be `stable`

Reply:
in sync with https://github.com/symfony/skeleton/blob/5.4/composer.json#L6

Reply:
note that we also have `"prefer-stable": true`

Reply:
but the **5.4** skeleton is meaningful about using dev stability as 5.4 is still in dev. Here, we talk about updating to *5.3* which is stable.

And `prefer-stable` might still select dev versions of other packages depending on the order of package selection (see https://github.com/composer/composer/issues/9917)

Reply:
yep sorry bad link, we use dev since 5.2:
https://github.com/symfony/skeleton/blob/5.2/composer.json#L6

Reply:
but is it an intended choice, or just forgetting to update the skeleton once the stable release happen ? I never remember discussing using that behavior for stable skeletons (I might have missed the discussion of course)